connectors/injected.ts

```diff
@@ -9,6 +9,7 @@ export default class Connector extends LockConnector {
         await window['ethereum'].enable();
       } catch (e) {
         console.error(e);
+        if (e.code === 4001) return;
```

Comment:
What does this fix?

Reply:
https://github.com/balancer-labs/snapshot/issues/177

Reply:
The reason for the bug was that the connect() continued after the user canceled. Then the $auth.isAuthenticated was true but there was no address. So the jizzicon picked a random image that it would show.